src/main/features/ccj/routes/confirmation.ts

```diff
@@ -13,6 +13,7 @@ export default express.Router()
       res.render(Paths.confirmationPage.associatedView,
         {
           defendantName: claim.claimData.defendant.name,
-          ccjRequestedAt: claim.countyCourtJudgmentRequestedAt
+          ccjRequestedAt: claim.countyCourtJudgmentRequestedAt,
+          reDetermination_requested_at: claim.reDeterminationRequestedAt
```

Comment:
To make it consistent with others please use camel case. i.e. rename `reDetermination_requested_at` to `reDeterminationRequestedAt `

Reply:
done